### PR TITLE
Website building on mac Version 2

### PR DESCRIPTION
**Pull request by hayleyZHL**
Deleted part 2 in README.

### CODE REVIEW

**Comment by david-yz-liu** [comment]
I don't think removing the note altogether is the right approach. Rather, why don't you just not show the output of `ls`, but rather just say "check that the file is executable for the current user". you can keep the `chmod` command too.

**Comment by hayleyZHL** [comment]
@david-yz-liu thanks, just committed the change.

File: README.md [hunk 1]
```diff
@@ -20,9 +20,8 @@ After this, you should be able to open `gen/index.html` and see the demo webpage
...
 1. For Mac users, open python file `includes.py` under `website/filters`. On the
    first line, change `#!/usr/bin/env python` to `#!/usr/bin/env python3`.
-2. In the command line, go to the website directory and run `ls -l filters` to
-   check if the file permission is executable. If you get `-rw-r--r--` in your
-   output, that means the file is read and write only. To make the file also
-   executable, run `chmod u+x filters/includes.py`. Now for a final check,
-   run `ls -l filters` again, if you get `-rwxr--r--` in the output, you've
-   successfully changed the file permission. Now try build the website again.
+
+2. Check if the file `website/filters` is executable for the current user. If
```
Comment (david-yz-liu) [inline README.md]: That's not the right file.

File: README.md [hunk 2]
```diff
@@ -20,9 +20,8 @@ After this, you should be able to open `gen/index.html` and see the demo webpage
...
+   it is not executalbe, run `chomod u+x filters/includes.py` in the command
```
Comment (david-yz-liu) [inline README.md]: You spelled "executable" wrong, and the command is also incorrect.

**Comment by david-yz-liu** [comment]
Thanks, @hayleyZHL!

**Pull request merged**